docs/user-guide/tof/dream.ipynb

```diff
@@ -414,7 +414,7 @@
     "table = lut_wf.compute(TimeOfFlightLookupTable)\n",
     "\n",
     "# Overlay mean on the figure above\n",
-    "table[\"distance\", 13].plot(ax=fig2.ax, color=\"C1\", ls=\"-\", marker=None)"
+    "table[\"data\"][\"distance\", 13].plot(ax=fig2.ax, color=\"C1\", ls=\"-\", marker=None)"
```

Comment:
Before reviewing further, I really wonder if TimeOfFlightLookupTable should be a (data)class instead of a `DataGroup`? What do you think?

Reply:
Or maybe even pydantic could be useful here, since it is about ser/deser.

Reply:
DataGroup makes it very easy to save to hdf5.
What do we gain from using a dataclass instead?

Reply:
> You can still convert to a DataGroup for ser/deser.

So you would do something like `sc.DataGroup(asdict(table)).save_hdf5('file.h5')` ?
Ok, I'll have a go at that.
I feel pydantic would be a bit overkill.

Reply:
Any good suggestions for a name for the field I named `"data"`?

Reply:
I'm going with 'array', I think it's quite common to use that for the thing that actually holds the data.